### ISSUE TITLE
Autocomplete entries

### ISSUE THREAD

**Issue by anthonydahanne** [canyapan/DietDiaryApp#8]
It's often tedious to have to fill in the same text over and over again.
Not mentioning that if the data is exported (through csv for example) you have the risk to have non consistent entries, that could be harder to process by an external system.

Here's my feature request : when the user inputs an entry description, have the DietDiaryApp autocomplete it , based on the existing entries.

Say, if I already added coffee as a description, whenever I type "co" the app would suggest me "coffee"

**Comment by canyapan** [comment]
I think, it will improve user experience. Therefore, I accept your request. I think, I will be able to fulfill it in September.

**Comment by anthonydahanne** [comment]
sounds great :-)
other idea though : to make things simpler on the autocomplete, change the description from a free text area to a list of lines.
For example, if I create breakfast  and as a description : 

    eggs
    bacon

I would rather create a description with 2 line entries : 

    eggs

and

    bacon


add have those several line entries support autocomplete
Maybe that's another feature request though.
Anyway, I don't know how much of an effort it is and how that could be compatible or not with the existing code...

**Comment by canyapan** [comment]
@anthonydahanne,
I have released a new version with auto completing entries from previous entries found within a month. Let's see how people react.

I thought about adding the entries in lines when I first implement this app. But, I decided not to do that it breaks the simplicity. And, now it is too hard to implement it, because people have already added their data and it is not possible to break down it into lines.
